<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeighingParameter</class>
 <widget class="QWidget" name="WeighingParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>652</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 40px &quot;Microsoft JhengHei UI&quot;;
color:rgb(255,255,255);
background:transparent</string>
       </property>
       <property name="text">
        <string>标定参数界面</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 40px &quot;Microsoft JhengHei UI&quot;;
color:rgb(255,255,255);
background:transparent</string>
       </property>
       <property name="text">
        <string>黄绿分界设定值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="le_canshu">
       <property name="minimumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#le_canshu{
border-image:url(:/按键_弹窗设置_大.png);
color:rgb(255, 255, 255);
font:30px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}
QPushButton#le_canshu:pressed{
background-image: url(:/按键_弹窗设置_按下_大.png);
color:rgb(255, 255, 255);
font: 14px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
	border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="le_weight">
       <property name="minimumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#le_weight{
border-image:url(:/按键_弹窗设置_大.png);
color:rgb(255, 255, 255);
font: 30px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}
QPushButton#le_weight:pressed{
background-image: url(:/按键_弹窗设置_按下_大.png);
color:rgb(255, 255, 255);
font: 14px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
	border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 40px &quot;Microsoft JhengHei UI&quot;;
color:rgb(255,255,255);
background:transparent</string>
       </property>
       <property name="text">
        <string>红黄分解点设定值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="le_ry">
       <property name="minimumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#le_ry{
border-image:url(:/按键_弹窗设置_大.png);
color:rgb(255, 255, 255);
font: 30px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}
QPushButton#le_ry:pressed{
background-image: url(:/按键_弹窗设置_按下_大.png);
color:rgb(255, 255, 255);
font: 14px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
	border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 40px &quot;Microsoft JhengHei UI&quot;;
color:rgb(255,255,255);
background:transparent</string>
       </property>
       <property name="text">
        <string>额定载重量</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="le_yg">
       <property name="minimumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>198</width>
         <height>46</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#le_yg{
border-image:url(:/按键_弹窗设置_大.png);
color:rgb(255, 255, 255);
font: 30px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}
QPushButton#le_yg:pressed{
background-image: url(:/按键_弹窗设置_按下_大.png);
color:rgb(255, 255, 255);
font: 14px &quot;Microsoft YaHei UI&quot;;
border:1px solid #979797;
	border-color: rgba(151, 151, 151, 1);
border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back_home">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#btn_back_home{
background-image: url(:/按键_返回主界面3.png);
border:none;
}
QPushButton#btn_back_home:pressed{

background-image: url(:/按键_返回主界面2.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#btn_back{
background-image: url(:/按键_上一页2.png);
border:none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
